mmv1/templates/terraform/examples/vertex_ai_featureonlinestore_with_beta_fields_bigtable.tf.erb

```diff
@@ -0,0 +1,23 @@
+resource "google_vertex_ai_feature_online_store" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  name     = "<%= ctx[:vars]['name'] %>"
+  labels = {
+    foo = "bar"
+  }
+  region   = "us-central1"
+  bigtable {
+   auto_scaling {
```

Comment:
Nit: inconsistent indentation. Note that these examples will be visible at the top of the documentation website for this resource.

Reply:
Updated the identation of all the examples with a terraform fmt.